contents/blog/a-chat-with-sid.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2020-08-03
+title: The early days of GitLab - a chat with Sid Sijbrandij
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+It gets pretty easy to idolize the superstars of tech. One of the coolest things we've learned is that many of the most successful founders will find time to talk with you. To put this in context, GitLab are one of the world's leading open core companies, worth \$2.75B. Kind of a big deal. We wanted to learn even more than we could discern from their handbook.
+
+[Sid Sijbrandij](https://twitter.com/sytses) is the founder and CEO, and has always been an inspiration to PostHog - GitLab's culture gave us the confidence to build PostHog all-remote, and it influenced the style of product we're building. Once we saw Sid had [liked a few of our releases](https://twitter.com/PostHogHQ/status/1288563434641072131) on Twitter, we thought we'd ask a friend of a friend to connect us, alongisde a suggested agenda of course.
```

Comment:
Is "alongside a suggested agenda" necessary? Maybe "...to connect us, hoping that we could set up a call" or something?